mysql-test/suite/rpl/r/rpl_mixed_ddl_dml.result

```diff
@@ -11,8 +11,8 @@ n
 2002
 connection master;
 show slave hosts;
-Server_id	Host	Port	Master_id
-2	127.0.0.1	9999	1
+Server_id	Host	Port	Master_id	Gtid_State_Sent	Gtid_State_Ack	Sync_Status
+2	127.0.0.1	9999	1	0-1-2		Asynchronous
```

Comment:
Have you though/discussed how/if represent "nil" of `Gtid_State_Ack`?
It feels the blank field is not a perfect way.  E.g `NULL` would do imo also to infer `Async...` of the status.